paymentsheet/src/test/java/com/stripe/android/utils/CompletableSingle.kt

```diff
@@ -0,0 +1,20 @@
+package com.stripe.android.utils
+
+import com.stripe.android.common.coroutines.Single
+import com.stripe.android.common.coroutines.asSingle
+import kotlinx.coroutines.flow.MutableStateFlow
+
+internal class CompletableSingle<T>(val value: T? = null) : Single<T> {
+    private val flow: MutableStateFlow<T?> = MutableStateFlow(value)
```

Comment:
Seems like this should use `CompletableDeferred` as an implementation detail, right?

Reply:
Ah yeah, that would be better